snorkel/utils.py

```diff
@@ -145,3 +146,124 @@ def tokens_to_ngrams(tokens, n_max=3, delim=' '):
     for root in range(N):
         for n in range(min(n_max, N - root)):
             yield delim.join(tokens[root:root+n+1])
+
+class SnorkelPrint:
+    def __init__(self, db_path):
+        self.conn = self.__create_connection(db_path)
+
+    @staticmethod
+    def __create_connection(db_path):
```

Comment:
nit: can we single underscore for private methods? re: https://www.python.org/dev/peps/pep-0008/#id36

Reply:
applies to private methods below

Reply:
Good point, just changed the names.